packages/gestalt-design-tokens/package.json

```diff
@@ -21,7 +21,8 @@
   "devDependencies": {
     "decomment": "^0.9.5",
     "style-dictionary": "^3.9.2",
-    "tinycolor2": "1.6.0"
+    "tinycolor2": "1.6.0",
+    "lodash.camelcase": "4.3.0"
```

Comment:
instead of adding a package (we are trying to deprecate lodash in Pinbaord...) can we just copy the function in a helper funcion file?

Reply:
I'm re-using a function from style-dictionary and they are using lo-dash.

This one is made from multiple files. This is a sub-package, so we are only importing that one function, not all of lo-dash. It's also a dev dependency, so it will have no impact on the bundle